app/authentication/authenticator.py

```diff
@@ -78,7 +78,7 @@ def _create_session_data_from_metadata(metadata):
         case_id=metadata.get('case_id'),
         case_ref=metadata.get('case_ref'),
         trad_as=metadata.get('trad_as'),
-        account_service_url=metadata.get('account_service_url'),
+        account_url=metadata.get('account_url'),
```

Comment:
Why are we changing this without updating the spec?
https://ons-schema-definitions.readthedocs.io/en/latest/respondent_management_to_electronic_questionnaire.html#required-fields